Machine_Learning/Bank Loan Approval Prediction/Web App/app.py

```
@@ -0,0 +1,147 @@
+    global predicted_result # global variable to get the prediction result
+    
+    prediction_Result = predicted_result[0][0] # stores predicted value (0 or 1)
+    probability = str(predicted_result[1][0]) # stores probability of 0 and 1
+    
+    accuracy = probability[1:-1].split()
+    rounded_numbers = [round(float(num), 3) for num in accuracy] # getting array of probabilities
+    
+    probability_0 = round(rounded_numbers[0]*100, 2)  # probability value for 0
+    probability_1 = round(rounded_numbers[1]*100, 2)  # probability value for 1
+    
+    result = 'Approved' if prediction_Result == True else 'Rejected'
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if prediction_Result:` for truth checks

Replace with `prediction_Result`

(E712)